src/components/WalletUnlock/ViewOnly.tsx

```diff
@@ -20,6 +20,13 @@ const ButtonWrapper = styled(Button)`
   margin-top: 4em;
 `;
 
+const Disclaimer = styled.p`
+  line-height: 24px;
+  padding-top: 16px;
+  text-align: center;
+  font-size: 18px;
+`;
```

Comment:
Rather than making a new comoponent for this, you can use [`Body`](https://github.com/MyCryptoHQ/MyCrypto/blob/master/src/components/NewTypography/Typography.tsx#L29).

https://github.com/MyCryptoHQ/MyCrypto/blob/master/src/features/AddAccount/components/NetworkSelectPanel.tsx#L57-L59